en/controllers/components.rst

```diff
@@ -146,7 +146,7 @@ controller, you could access them like so::
 
         public function delete()
         {
-            if ($this->Post->delete($this->request->data('Post.id')) {
+            if ($this->Post->delete($this->request->getData('Post.id')) {
```

Comment:
Same for authentication.rst, cookie.rst, csrf.rst, pagination.rst, request-handling.rst, request-response.rst
etc... e.g. all the changes here seem to miss versionadd or versionchanged flags.

And: Can we teach sphinx to echo out css classes for the version blocks? If we can do, it would be easy to hide/show blocks based on a dropdown filter (e.g. show 3.3 docs would hide all versionadd blocks for 3.4+)